api/db/migrations/20240226050159_object_add_texture_field.ts

```diff
@@ -0,0 +1,24 @@
+import { Knex } from 'knex';
+
+const columnName = 'texture';
```

Comment:
The mall staff needs to be able to view the texture file to make sure there is nothing inappropriate or that it is not copyright infringement. This is needed to be saved in the db because the filename of the texture file is not changed like the wrl file and the thumbnail.